routes/api.php

```diff
@@ -13,3 +17,15 @@
 
 $router->post("/login", [AuthenticationController::class, "login"]);
 $router->post("/register", [AuthenticationController::class, "register"]);
+
+$router->group([
+    "middleware" => "auth:sanctum",
+], function () use ($router): void {
```

Comment:
Type router.

Reply:
I have problem with that..
Can I type it?

Reply:
Is Illuminate\Routing\Router type not sufficient?